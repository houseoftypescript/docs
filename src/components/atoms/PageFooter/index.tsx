import Container from '@mui/material/Container';
import React from 'react';

export type PageFooterProps = {
  children?: React.ReactNode;
};

const PageFooter: React.FC<PageFooterProps> = ({ children = '' }) => {
  return (
    <footer className="border-t">
      <Container className="py-8">{children}</Container>
    </footer>
  );
};

PageFooter.displayName = 'PageFooter';
PageFooter.defaultProps = { children: '' };

export default PageFooter;
