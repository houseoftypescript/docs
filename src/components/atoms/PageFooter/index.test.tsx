import PageFooter from '.';
import { render } from '@testing-library/react';

describe('PageFooter', () => {
  test('render default', () => {
    const wrapper = render(<PageFooter />);
    expect(wrapper.container).toMatchSnapshot();
  });
});
