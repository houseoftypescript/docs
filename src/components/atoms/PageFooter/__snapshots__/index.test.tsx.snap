// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageFooter render default 1`] = `
<div>
  <footer
    class="border-t"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg py-8 css-1oqqzyl-MuiContainer-root"
    />
  </footer>
</div>
`;
